src/ome_zarr_models/v04/image.py

```diff
@@ -18,6 +19,9 @@ class Image:
 
     def __init__(self, group: zarr.Group):
         self._attrs = MultiscaleGroupAttrs(**group.attrs.asdict())
+        if "labels" in group:
+            if not isinstance(group["labels"], zarr.Group):
+                raise ValidationError("The special group 'labels' is not a zarr group")
```

Comment:
this error message is a bit cryptic -- maybe better to say that a group was expected, but an array was found instead?